cli/src/cli.rs

```diff
@@ -5,7 +5,6 @@ use crate::{
     nonce::{self, *},
```

Comment:
@CriesofCarrots I'm pulling out the storage program and archives, did I miss anything here?

Reply:
CLI and RPC look good to me!